apps/admin/frontend/src/api.ts

```diff
@@ -398,18 +398,26 @@ export const getCvrWriteInImageViews = {
       ? ['getCvrWriteInImageViews', input]
       : ['getCvrWriteInImageViews'];
   },
-  useQuery(input: GetCvrWriteInImageViewsInput, options = { enabled: true }) {
+  useQuery(input?: GetCvrWriteInImageViewsInput) {
     const apiClient = useApiClient();
     return useQuery(
       this.queryKey(input),
       () =>
         apiClient.getCvrContestWriteInImageViews({
-          cvrId: input.cvrId,
-          contestId: input.contestId,
+          cvrId: input?.cvrId || '',
+          contestId: input?.contestId || '',
```

Comment:
nit: prefer `??` unless there's a special need for falling back from a falsy value.